tests/browser/test/procedure_test.js

```diff
@@ -82,7 +82,7 @@ suite('Testing Connecting Blocks', function (done) {
       'Text',
       'text_print'
     );
-    await printFlyout.dragAndDrop({x: 50, y: 20});
+    await printFlyout.dragAndDrop({x: 50, y: 0});
```

Comment:
It might be good to use the coordinate conversions to drop blocks from the flyout at a specific workspace location, rather than dragging them by a relative amount in pixels.